ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.14.yaml

```diff
@@ -0,0 +1,225 @@
+base_images:
+  rhel-8-golang-1.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- dockerfile_path: Dockerfile
+  from: rhel-8-golang-1.20
+  to: oadp-operator
+- dockerfile_path: build/ci-Dockerfile
+  from: rhel-8-golang-1.20
+  to: test-oadp-operator-1.3
+operator:
+  bundles:
+  - dockerfile_path: build/Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/konveyor/oadp-operator:oadp-1.3
+    with: oadp-operator
+promotion:
```

Comment:
here I would change to be like here https://github.com/openshift/release/blob/master/ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.14.yaml#L24-L29

The name is like the the image name, so would let it like in the linked. Otherwise, each oadp version will have a different image, instead of just a different tag

Reply:
Did I get this right? [commit](https://github.com/openshift/release/pull/45664/commits/604f2ce4f43545eaf0d0b182967d7567021e61f1)

Reply:
Almost

diff https://github.com/openshift/release/blob/master/ci-operator/config/openshift/velero/openshift-velero-konveyor-dev.yaml
and https://github.com/openshift/release/blob/master/ci-operator/config/openshift/velero/openshift-velero-oadp-1.2.yaml

and you'll see name does not have version, just the additional_images part.

Reply:
Which produces `registry.ci.openshift.org/konveyor/velero:velero-oadp-1.2` seen [here](https://github.com/kaovilai/release/blob/master/core-services/image-mirroring/konveyor/mapping_konveyor_latest#L20C1-L20C58)

Reply:
The current promotion would produce IIUC `registry.ci.openshift.org/konveyor/oadp-operator-1.3/oadp-operator` which is anti-pattern.

Reply:
summarizing what @kaovilai commented, the change needed
```yaml
  additional_images:
    oadp-operator-1.3: oadp-operator-1.3
    test-oadp-operator-1.3: test-oadp-operator-1.3
  name: oadp-operator
```